### ISSUE TITLE
Multi-touch support for virtual keyboard

### ISSUE THREAD

**Issue by eimaster** [openMSX/openMSX#1047]
When using openMSX on-screen keyword I can't type the shifted characters. Yes SHIFT key works but it works alone! In real MSX keyboard, If I want to type the character "!" then I press SHIFT+"1" but I can't do that in openMSX on-screen keyboard. I think the problem is that openMSX on-screen keyboard does not support multi-touch!

**Comment by MBilderbeek** [comment]
Indeed, it does not support multi-touch (yet), but you can still hold shift and then type the 1 key. Use button B to toggle a key to pressed down status. It will get a special color to indicate this.
